### ISSUE TITLE
Remove Doctrine mappings YML from documentation

### ISSUE THREAD

**Issue by cosminsandu** [symfony/symfony-docs#20050]
Since Doctrine 3.0 does not support YML configuration (see https://github.com/doctrine/DoctrineBundle/issues/1071 https://github.com/doctrine/orm/blob/3.0.x/UPGRADE.md#bc-break-remove-yaml-mapping-drivers https://github.com/doctrine/orm/blob/3.0.x/UPGRADE.md#deprecated-yaml-mapping-drivers ) Doctrine Configuration Reference (DoctrineBundle) documentation > [Mapping Configuration](https://symfony.com/doc/current/reference/configuration/doctrine.html#mapping-configuration) > [type](https://symfony.com/doc/current/reference/configuration/doctrine.html#type) should be updated to not contain `yml` format anymore.

please let me know if it is ok to remove the `yml` option from documentation.

**Comment by cosminsandu** [comment]
I think it should be a deprecated message - as in case of Annotations ([on version 6.4](https://symfony.com/doc/6.4/reference/configuration/doctrine.html#type) )

![image](https://github.com/user-attachments/assets/29bce984-6b5e-46ef-87fe-8918229daf1f)

Please let me know on which Symfony version should I add this change.

**Comment by xabbuh** [comment]
We can do the change in the `5.4` branch as Doctrine ORM 3 is usable with Symfony 5.4.

**Comment by cosminsandu** [comment]
Looks like I need to add `.. deprecated::` to a version of Symfony but in this case it is deprecated in ORM 3.0
```
Error: Please provide a numeric version behind ".. deprecated::" instead of "ORM 3.0"
Error: Please provide a numeric version behind ".. deprecated::" instead of "ORM 3.0"
Error: Please only provide ".. deprecated::" if the version is greater/equal "5.0"
```
https://github.com/symfony/symfony-docs/actions/runs/10006460548/job/27659228920?pr=20052

Please advice

**Comment by javiereguiluz** [comment]
Closing as fixed in #20052.
